docs/user-guide/CAI.rst

```diff
@@ -0,0 +1,308 @@
+.. include:: /content/nemo.rsts
+
+.. _model-aligner-cai:
+
+Constitutional AI: Harmlessness from AI Feedback
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+
+`Constitutional AI (CAI)`_ is an approach by Anthropic to train AI systems that are helpful, honest, and harmless, without relying on human feedback labels for harmlessness.
+CAI uses a set of natural language principles to guide AI behavior and self-improvement.
+The method consists of two stages: a supervised learning stage, where the AI critiques and revises its own responses according to the principles, and a reinforcement learning stage, where the AI learns from its own feedback based on the principles.
+CAI allows training a harmless but non-evasive AI assistant that engages with harmful queries by explaining its objections to them.
+
+.. _Constitutional AI (CAI): https://arxiv.org/abs/2212.08073
+
+CAI
+###############
+The basic steps of CAI, as illustrated in Figure 1 of the paper (below):
```

Comment:
The basic steps of CAI are described in this section and illustrated in the figure below.

Reply:
Thanks, revising to: 
"The basic steps of CAI are described in this section and illustrated in the figure below (Figure 1)."

Reply:
There are two figures in this file. If you want to refer to a figure by a number, you need to edit the figure title and add Figure 1. "<Title>" and Figure 2. "<Title>". I suggest  not adding figure numbers in this file as there are only two figures in this file and it is not necessary. 

Instead, retain my original edit: The basic steps of CAI are described in this section and illustrated in the figure below.